test/integration/targets/azure_rm_appgw/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/azure
+destructive
+posix/ci/cloud/group2/azure
```

Comment:
The alias was just recently changed, see https://github.com/ansible/ansible/blob/devel/test/integration/targets/azure_rm_loadbalancer/aliases for the new naming standard for this field https://github.com/ansible/ansible/blob/devel/test/integration/targets/azure_rm_loadbalancer/aliases